packages/icons/web/v2/pix_32.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Shouldn't the icon be square?

Reply:
No, that's a new icons pack. This is right